app/controllers/player_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
       @slide.increment(:embedded_view).increment(:total_view).save
     end
     @start_position = slide_position
-    s = render_to_string layout: 'plain', collection: @slide
+    s = render_to_string layout: 'javascript', collection: @slide
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 's' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->